source/_integrations/zone.markdown

```diff
@@ -10,15 +10,58 @@ ha_codeowners:
   - '@home-assistant/core'
 ha_domain: zone
 ha_integration_type: system
+related:
+  - docs: /docs/configuration/basic/
+    title: Editing basic settings
+  - docs: /docs/organizing/areas/
+    title: Areas
+  - docs: /getting-started/onboarding/
+    title: Onboarding
 ---
 
-Zones allow you to specify certain regions on earth (for now). When a device tracker sees a device to be within a zone, the state will take the name from the zone. Zones can also be used as a [trigger](/docs/automation/trigger#zone-trigger) or [condition](/docs/scripts/conditions/#zone-condition) inside automation setups.
+Zones allow you to specify certain regions on a map. When a device tracker sees a device to be within a zone, the state will take the name from the zone. Zones can also be used as a [trigger](/docs/automation/trigger#zone-trigger) or [condition](/docs/scripts/conditions/#zone-condition) inside automations. For example, for showing the weather, opening the shades at sunrise, or starting the vacuum when you leave the home.
 
-Zones can be added and managed through the user interface at {% my zones title="**Settings** > **Areas, labels & zones**" %}.
+## About the home zone
 
-![Screenshot of the UI for adding or editing a zone](/images/integrations/zone/zone_edit_ui.png)
+During the [onboarding process](/getting-started/onboarding/), Home Assistant asked for your home location. You either entered this manually or asked Home Assistant to detect it by IP address geolocation.
 
-You can add a zone in the user interface by specifying the GPS coordinates or dragging the icon on the map. You can adjust the zone radius (except for the Home zone) by changing the size of the zone circle.
+During onboarding, this location was used to create the home zone with a 100&nbsp;m radius. The home zone is a special, pre-defined zone with a few characteristics that set it apart from other zones.
+
+- The name of this zone is defined by the name of your (which defaults to "Home").
+- The home zone cannot be deleted and is designated with the home icon in the zone configuration page.
+- The home zone's location is used by integrations that are location-based. For example, the [Sun integration](/integrations/sun/), which uses it to calculate the position of the sun relative to your home.
+- During onboarding, Home Assistant defined a unit system and time zone based on this location. If you change the location later, unit system and time zone will not be changed automatically anymore.
+- Devices that are in the home zone will not appear on the map in the Home Assistant UI. If you are using your phone for example for presence detection and location based automations, don't be alarm if you don't see your phone on the Map dashboard while you are at home.
+
+## Editing your home zone
+
+1. To edit your home zone, go to {% my general title="**Settings** > **System** > **General**" %}.
```

Comment:
This can now just be edited under the zones itself. There is no longer a need to handle different steps for the home zone.

Reply:
well, from what I see, using the current dev version, the handling is a bit different for the home zone. 
- You cannot change the name of the home zone from the Zone page. you need to change the name of the zone. This is why I chose this as the entry point.
- Also, you cannot change the icon of the home zone. 
So, I don't think reusing the same procedure would be entirely accurate.

Reply:
Honestly, I do not agree. We make the home zone editable via the zones page, the have a more unified and expected experience.

I think is it more valuable to learn the user where zones can be managed in general instead of documenting the weird exception.

Yes, the Home zone has limitations, but those can be described (and should be described anyways)